.env.example

```
@@ -1,2 +1,2 @@
-EFA_SKIP_FULL_HASHLIST_UPDATE=false # Whether to skip hashlist file generation
-EFA_DEFAULT_WORKSPACE_HASHLIST=/develop/eve-fit-assistant/data/resources/tranquility/output/hash_list.txt # Default path to the hashlist file, for dev-usage
+EFA_SKIP_FULL_MANIFEST_UPDATE=false # Whether to skip snapshot manifest generation
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)